fe/bar.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import jax.numpy as jnp
+from jax.scipy.special import logsumexp
+import pymbar
+import jax
+
+def EXP(w):
+    T = float(jnp.size(w))
+    deltaF = - (logsumexp(-w) - jnp.log(T))
+    return deltaF
+
+def BARzero(w, deltaF):
```

Comment:
Please a docstring for what the parameters are and their expected shapes. We typically use numpydoc:

https://numpydoc.readthedocs.io/en/latest/format.html